### ISSUE TITLE
 Nice work,When would you update the model eval code？

### ISSUE THREAD

**Issue by Liusandian** [leohsuofnthu/Tensorflow-YOLACT#8]
Nice to see your job for complete the Yolact method,I want to know when it will be updated for the model eval code，I am looking forward to your reply,thanks.

**Comment by leohsuofnthu** [comment]
Hi, Thanks for your attention. I just finish optimizing the training speed today. I will start training soon. I think the evaluation part will be updated within this week. :)

**Comment by Nihar1989** [comment]
Hi,your resource has been of great help ,when can you update the inference part,also can this model be converted into tensorflowlite

**Comment by Liusandian** [comment]
> Hi,your resource has been of great help ,when can you update the inference part,also can this model be converted into tensorflowlite



> Hi,your resource has been of great help ,when can you update the inference part,also can this model be converted into tensorflowlite

I have tried ,this model impements from custom Subclass Model,cannot covert into tflite.If you want convert it,you need to rewrite the model in tf Keras API style

**Comment by JackSkyz** [comment]
Hello, I want to congratulate you on your work, complete and well readable.
I have been training a network, and now I would like to get some predictions, so I want to ask you, when could the inference code be updated?

**Comment by leohsuofnthu** [comment]
Hello friends,
Thanks for noticing my work. I am sorry that recently I am focusing on my course projects, so the update here might be slow. I wrote a script for quickly testing your training result on valid dataset in "eval" branch (https://github.com/leohsuofnthu/Tensorflow-YOLACT/blob/eval/test/test_detection.py) you can directly run this with your training checkpoints to see some detection results before the full release of inference and evaluation. the segmented masks drawing will update soon. Besides, I just found that some of my settings of data augmentation are not correct, which cause some issue of my reproduced results on COCO, my goal is to finish those before May. Thanks again.

**Comment by JackSkyz** [comment]
It's awesome, to be your first object detection implementation project you do really well

**Comment by Liusandian** [comment]
> Hello friends,
> Thanks for noticing my work. I am sorry that recently I am focusing on my course projects, so the update here might be slow. I wrote a script for quickly testing your training result on valid dataset in "eval" branch (https://github.com/leohsuofnthu/Tensorflow-YOLACT/blob/eval/test/test_detection.py) you can directly run this with your training checkpoints to see some detection results before the full release of inference and evaluation. you can see the result as below(red for the label, blue for the prediction)
> ![image](https://user-images.githubusercontent.com/42263429/78074172-3b800b00-7370-11ea-8ce6-1356c21386a7.png)
> , and the segmented masks drawing will update soon. Besides, I just found that some of my settings of data augmentation are not correct, which cause my reproduced results on COCO is not ideal for now, my goal is to finish those before May. It is my first Objection detection implementation, I will do my best. Thanks again.

Thanks for your reply,I tried to use your verification code to run the model inference, but the detection result  is not correct,some bbox is redundant, so I tried to uncomment the fast nms module code, but the code is wrong again, Do you have a fast nms function which  can run  the fast nms steps?
